ai-platform-unified/notebooks/unofficial/reduction_server/distributed-training-reduction-server.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Branding: Only use Vertex AI when referring to the service/framework itself. Otherwise, drop the AI</p><p> </p><p>Vertex AI custom job -&gt; Vertex Custom Training job</p><p> </p><p>Branding: Google Cloud Storage -&gt; Cloud Storage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>